backend/services/ContentPublisherService.ts

```diff
@@ -1,10 +1,11 @@
 import {
   Client as ContentPublisherClient,
   type Components,
-} from "../types/openapi-content-publishing-service.js";
-import openapiJson from "../openapi-content-publisher.json" assert { type: "json" };
+} from "../types/openapi-content-publishing-service";
+import openapiJson from "../openapi-specs/content-publishing-service.json" assert { type: "json" };
```

Comment:
If we've updated to TypeScript v5.3+:

```suggestion
import openapiJson from "../openapi-specs/content-publishing-service.json" with { type: "json" };
```

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-3.html#import-attributes